### DISCUSSION TITLE
Option to make read only to any particular dropdown

### DISCUSSION THREAD

**Discussion by alok01rai** [xrutayisire/react-js-cron#21] [Q&A] [answered]
Hi,

I would like to know how I could make the 2nd & 3rd dropdown as read only.  Means I would like Month, Year should be always * or lets say I would like to keep all dropdown as single select. 

Any help will be appreciable.

**Comment by xrutayisire** [comment]
Hello,

It could be a new feature. Do you want to have the period dropdown without the month and year period?

**Reply by alok01rai** [reply]
Single select for all dropdowns will work for my current requirement.

**Reply by xrutayisire** [reply]
Hum, sorry I don't understand, what do you mean by single select?

**Comment by alok01rai** [comment, marked as answer]
I mean right now if you click on any dropdown we can select multiple option i. e.  If week, I can select monday, tuesday (more than 1 option in week), same goes for days, hours, minutes. 

I am looking for an option where visitor can select onlybone option from each dropdown. I. E.  1 month,  1 date,  1 hour, 1 minute.

**Reply by alok01rai** [reply]
Like in single select dropdown we have option to select only one value.

**Reply by xrutayisire** [reply]
Oh ok I understand :) So it's still possible to select the month but only one, same for other select. 

Period don't change so, right?

I will see what I can do, but it seems easy to add this, and since it will be a new prop it will not be a breaking change. I can do it quickly if I don't face any problem.

Can you create a new issue, with this feature request?

**Comment by alok01rai** [comment]
So it's still possible to select the month but only one, same for other select

-- Yes

Period don't change so, right?
-- Correct
